backend/src/modules/Announcements/service/Announcements.service.js

```diff
@@ -1,24 +1,53 @@
 import Announcement from './Announcement.js';
```

Comment:
Błędy mógłbyś w serwerze wyrzucać, jeśli potrzebujesz :)

Reply:
Po repository raczej nie spodziewamy się, że błędami będzie siać, ale już po servisie może być to pożądane zachowanie.
Literka "L".

Reply:
Dodatkowo tutaj przez errory w Repo to Repo kontaktuje się z Controller z pominięciem Service, a my chcemy rozłączyć Repo i Controller

Reply:
+1

Reply:
done